styles.css

```diff
@@ -0,0 +1,293 @@
+}
+
+/* struggling with centering content in the middle of the box!*/
+
+.about-content {
+  vertical-align: middle;
+  line-height: 2;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#schedule {
+  width: 440px;
+  height: 440px;
+  text-align: center;
+  border-radius: 50%;
+  border-style: solid;
+  border-width: 5px;
+  border-color: purple;
+  background-color: purple;
+  color: white;
+  margin: 10px;
+  display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      padding: 65px;
+}
+
+div.schedule-date {
+  font-weight: bold;
+  color: white;
+}
+div.schedule-content {
+  font-size: 15px;
+  text-align: center;
+  color: white;
+  margin: 0px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#speakers {
+  width: 350px;
+}
+.speakers-card {
+  text-align: justify;
+  width: 350px;
+  padding-left: 18px;
+}
+.speaker-card {
+  text-align: justify;
+  margin-bottom: 24px;
+  padding: 10px;
+  border-radius: 25px;
+  text-align: left;
+  border-style: solid;
+  border-width: 5px;
+  border-color: black;
+  width: 450px;
+  height: 300px;
+  overflow: scroll;
+  display: inline-block;
+}
+img.speaker-photo {
+  float: left;
+  width: 200px;
+  height: 275px;
+  object-fit: cover;
+  padding: 15px;
+}
+#talks {
+  width: 350px;
+}
+.talks-card {
+  list-style: inline;
+  text-align: justify;
+  list-style: none;
+  border-top: 3px solid;
+  border-radius: 25px;
+  text-align: left;
+  border-style: solid;
+  border-width: 5px;
+  border-color: black;
+  width: 450px;
+}
+img.author-photo {
+  width: 400px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+footer.form {
+  border-radius: 25px;
+  text-align: justify;
+  border-style: solid;
+  border-width: 3px;
+  border-color: black;
+  padding: 20px;
+  width: 350px;
+  height: 220px;
+}
+
+@media (min-width: 520px) {
+  header {
+    width: 600px;
+    height: 600px;
+    position: relative;
+    left: 100px;
+  }
+
+  /* how do you determine hierarchy?????? why does the buy
+  ticket sometimes go over and under the other elements*/
+
+  .buy-tickets-button:link {
+    position: fixed;
+    top: 200px;
+    right: 180px;
+  }
+
+  nav {
+    text-align: center;
+    vertical-align: middle;
+    line-height: 1px;
+    font-size: 18px;
+    color: white;
+  }
+  #about {
+    width: 560px;
+    height: 560px;
+    position: relative;
+    left: 630px;
+  }
+
+  #schedule {
+    position: relative;
+    left: 200px;
+    width: 600px;
+    height: 600px;
+    padding: 15px;
+  }
+}
+
+@media (min-width: 960px) {
+  .speaker-card {
+    display: inline-block;
+    vertical-align: top;
+  }
+
+  .talks-card{
+    display: inline-block;
+    vertical-align: top;
+  }
+
+  /*element.style {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+  }*/
+}
```

Comment:
I think there's some basic organization and tidying up that would help your CSS:

1. Make sure all your base elements are styled first, then class names, then media queries.
2. Avoid using HTML + class selectors like `div.class-name` since it only serves to complicate your CSS.
3. When possible, use either shared classes or combined selectors (like `header, #about, #schedule`) to avoid rewriting/duplicating CSS.
4. Make sure that in your pseudoclasses (like `:visited` and `:hover`) and your media queries you're only writing the CSS you need to -- not repeating all styles.

Overall, I think you've attempted a complicated and fun layout and gotten very close to nailing it, which is extra impressive given the limited CSS we covered in this course. 🏆 Great work!